### PR TITLE
Automated test for checking the input for duplicated pipeline run

### PR DESCRIPTION
**Pull request by diegolovison**
rhods-smoke/4605

### CODE REVIEW

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 1]
```diff
@@ -147,18 +147,28 @@ Open Pipeline Detail
...
+    IF    ${present} == ${FALSE}
+        Wait Until Page Contains Element    xpath=//button[text()='Data Science Pipelines']
+        Click Element    xpath=//button[text()='Data Science Pipelines']
+        Wait Until Page Contains Element    xpath=//a[text()='Runs']
+    END
     Click Element    xpath=//a[text()='Runs']
-    Wait Until Page Contains Element    xpath=//span[text()='Triggered']
+    IF    ${force_reload} == ${TRUE}
+        Reload Page
+    END
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:163]: is it for workarounding any product issue?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 2]
```diff
@@ -147,18 +147,28 @@ Open Pipeline Detail
     Capture Page Screenshot
 
 Open Pipeline Run
-    [Documentation]    Open the Pipeline Run detail
-    [Arguments]    ${pipeline_run_name}
-    Wait Until Page Contains Element    xpath=//button[text()='Data Science Pipelines']
-    Click Element    xpath=//button[text()='Data Science Pipelines']
-    Wait Until Page Contains Element    xpath=//a[text()='Runs']
+    [Documentation]    Open the Pipeline Run detail. The menu hold a state and if you click twice in the same element,
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:150]: is there a ticket for this bug?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource (hunk 1 above)
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:163]: you click on Runs and nothing happens right? hence you need the page reload, is that correct?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 3]
```diff
@@ -178,3 +190,22 @@ Verify Pipeline Run Is Completed
     Open Pipeline Run    ${pipeline_run_name}
     Wait Until Page Contains Element    //span[@class='pf-v5-c-label__text' and text()='Completed']    timeout=10m
     Capture Page Screenshot
+
+# robocop: disable:line-too-long
+Open Pipeline Run Duplicate Page
+    [Documentation]    Open only the Pipeline Run Duplicate Page
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:196]: what does it mean "only"? what else could be opened?

File: ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot [hunk 4]
```diff
@@ -173,3 +175,10 @@ Verify Pipeline Run Deployment Is Successful    # robocop: disable
     ${containerStatuses}=  Create List        terminated    terminated
     ...    terminated    terminated    terminated
     Verify Deployment    ${valid_model}  1  1  ${containerNames}    ${podStatuses}    ${containerStatuses}
+
+Verify Data Science Parameter From A Duplicated Run Are The Same From The Compiled File
+    [Documentation]    Verify Data Science Parameter From A Duplicated Run Are The Same From The Compiled File
+    ${input_parameters}=    Open Pipeline Run Duplicate Page    ${PIPELINE_TEST_RUN_BASENAME}
```
Comment (bdattoma) [inline ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot:181]: sounds this kw should be more named sth like "get input parameters" rather then `Open ..`

**Review by nehachopra27 — APPROVED** [review]
LGTM

**Pull request merged**